src/client/datascience/commands/notebookCommands.ts

```diff
@@ -35,13 +35,20 @@ export class NotebookCommands implements IDisposable {
             this.commandManager.registerCommand(Commands.NotebookEditorCollapseAllCells, this.collapseAll, this),
             this.commandManager.registerCommand(Commands.NotebookEditorExpandAllCells, this.expandAll, this),
             this.commandManager.registerCommand(Commands.NotebookEditorKeybindSave, this.keybindSave, this),
-            this.commandManager.registerCommand(Commands.NotebookEditorKeybindUndo, this.keybindUndo, this)
+            this.commandManager.registerCommand(Commands.NotebookEditorKeybindUndo, this.keybindUndo, this),
+            this.commandManager.registerCommand(Commands.NotebookEditorToggleOutput, this.toggleOutput, this)
         );
     }
     public dispose() {
         this.disposables.forEach((d) => d.dispose());
     }
 
+    private toggleOutput() {
+        if (this.notebookEditorProvider.activeEditor) {
+            this.notebookEditorProvider.activeEditor.toggleOutput();
```

Comment:
I was under the impression this was supposed to be per-cell, not for all cells in the entire notebook document.

Reply:
@claudiaregio is it per cell or the whole document?

Reply:
In Jupyter classic 'o' is per cell

Reply:
It will flip all selected cells

Reply:
Jupyter classic actually applies to all selected cells, so this should toggle output for all selected cells in the case of multiselect. 
![Jupyter o shortcut](https://user-images.githubusercontent.com/19276747/114650467-26360000-9c97-11eb-82e1-fc7b5f1c46e8.gif)